languages/wikipron/src/whitelist.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
```

Comment:
Put a documentation string here explaining what this does in a single sentence, with a period at the end,